### ISSUE TITLE
Order Book Price Units Should Be Inversed

### ISSUE THREAD

**Issue by RhettCreighton** [stellar/stellar-client#1100]
Shouldn't it be 222222 STR / BTC, not 222222 BTC / STR?

![screenshot](https://cloud.githubusercontent.com/assets/2388024/5152360/783b2114-71a6-11e4-99a5-dd9fb09b1ff0.png)

**Comment by deckar01** [comment]
I agree that the label should match the calculation. It was originally in the suggested order until https://github.com/stellar/stellar-client/issues/1035.

**Comment by bekkibolthouse** [comment]
If you think of the currency pairs as a single unit, where the base currency is first and the quote currency comes second (BASE/QUOTE). When you look at the trading client with this lens, you see that when you set your trading pairs, the currency that you intend to buy always displays as the BASE.  So any offer to buy STR will read as "STR/QUOTE" and offers to buy BTC will read as "BTC/QUOTE." When you look at the order book, you see the spread in price between offers to buy the BASE (bids) and the offers to sell the BASE (asks). The forward slash shouldn't be interpreted as a math function, more like grammar or punctuation.
